hello-vulkan/README.md

```diff
@@ -9,8 +8,27 @@ Aside from the base functionality the source code also covers convenient feature
 - Vulkan pre-rotation 
 to enable fast and seamless screen rotation implemented on the application side. You can read more about it here:
 https://developer.android.com/games/optimize/vulkan-prerotation
+  
+Pre-requisites
+--------------
+- Android Studio 4.2+ with [NDK](https://developer.android.com/ndk/) bundle.
 
-Validation layer
+Getting Started
+---------------
+1. [Download Android Studio](http://developer.android.com/sdk/index.html)
+1. Launch Android Studio.
+1. Open the sample directory.
+1. Open *File/Project Structure...*
+- Click *Download* or *Select NDK location*.
+1. Click *File/Sync Project with Gradle Files*.
+1. Click *Run/Run 'app'*.
+
+Screenshots
+---------------
+![Vulkan Triangle example](documentation-assets/example-triangle.png)
+
+
+Validation layers
```

Comment:
this is still using "hellovk/..", should be updated with '**hello-vulkan**/app/src/...' right? might be good to search whole codebase for similar references.

Reply:
to be fair, I'll just remove any reference to the parent folder since this readme will be placed within the hello-vulkan directory.